extendable_fastapi/models.py

```diff
@@ -0,0 +1,20 @@
+from extendable_pydantic import ExtendableBaseModel
```

Comment:
To follow the naming conventions of `odoo.addons.fastapi`, we should perhaps define this model in `schemas.py` :thinking:

Reply:
Done